zebra-state/src/service/finalized_state.rs

```diff
@@ -503,6 +539,14 @@ impl FinalizedState {
             .expect("note commitment tree must exist if there is a finalized tip")
     }
 
+    /// Returns the Orchard note commitment tree of the finalized tip
+    /// or the empty tree if the state is empty.
```

Comment:
Confusing typo:
```suggestion
    /// or `None` if the state is empty.
```

Reply:
I managed to get the entire comment wrong because I copied from the other method :grimacing: Thanks for catching it. Fixed in 949a5e5c